blog-homepage/src/Picture.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
This component is just a wrapper for a `<img />` tag and you probably don't need it? 🤷